Platform/Samsung/sdm845/starqlte.fdf.inc

```diff
@@ -0,0 +1,6 @@
+// per-device BSP DXEs
+
+// ACPI Tables
+
+// Mainline device tree blob
+
```

Comment:
enable linux-simple-mass-storage but no mainline device tree blob. you need add mainline device tree blob or disable linux-simple-mass-storage

Reply:
should be good now?